lib/token_store.rb

```diff
@@ -52,8 +52,8 @@ def self.cache_store_options(namespace, token_ttl)
     MiqMemcached.default_client_options.merge(
       {
         :namespace  => "MIQ:TOKENS:#{namespace.upcase}",
-        :threadsafe => true,
-        :expires_in => token_ttl
+        :expires_in => token_ttl,
+        :pool_size  => 10,
```

Comment:
threadsafe is now specified above, so just add the default pool size of 10 for token store.

Reply:
Same question here...if default has 10 why do we re-specify it?

Reply:
Same as above... closing this conversation as it's the same discussion.